src/models/connectionCredentials.ts

```diff
@@ -1,7 +1,7 @@
-/* --------------------------------------------------------------------------------------------
- * Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
It would be great in the future to try and keep unrelated changes like these in another PR...sending the header fixes out before hand and getting that in beforehand would help reduce the randomization when reviewing this

Reply:
I had this stuff initially but like always the PR kept growing since I don't hear anything. I would commit in parts if I got reviews

Reply:
Sure - although from my point of view if I see a PR go out that keeps having more commits added to it I'm going to assume it's not finished yet so tend to hold off looking at it until I hear it's ready. What I usually do if I have more work I want to continue doing is then create another branch off of the initial branch and continue doing my work there so the open PR isn't cluttered with later changes - then when that PR is merged in you can rebase the newer branch to master and continue working as normal from there. It makes the whole reviewing process a lot more straightforward and easier to keep track of for those involved. 

---
In reply to: [321911026](https://github.com/microsoft/vscode-mssql/pull/1282#discussion_r321911026) [](ancestors = 321911026)

Reply:
@abist you should prefer to create different PRs for different changes rather than growing an existing PR.  If you need new changes based on pending changes then you can create the new PR off your dev branch and diff it against that for concurrent review.  Also, if you're not getting tracking on a review please send email or set up a code review meeting to drive to completion.  Thanks!